### ISSUE TITLE
Opcodes documentation location

### ISSUE THREAD

**Issue by oliverearl** [php/doc-en#1723]
Hi all,

Apologies if this is an inappropriate place to ask, but I've not had any success finding it on my own so far. Previously, there was documentation available at [https://secure.php.net/manual/internals2.opcodes.php](https://secure.php.net/manual/internals2.opcodes.php) that provided an outline of the Zend Engine opcodes. 

This link no longer works, and as it's referenced in [PHP The Right Way](https://github.com/codeguy/php-the-right-way/pull/918) I'd like to update to it a modern official reference, if there is one? 
Currently my pull request proposes a mirrored source hosted by Zend.

tl;dr Does this documentation (or a modern equivalent) still exist? Where can I find it?

**Comment by cmb69** [comment]
The "Hacker's guide" book in the PHP manual was grossly outdated, and since the doc-team lacks time and knowledge of these internals, it was decided to remove it. The plan is to add still relevant info to https://www.phpinternalsbook.com/. For now, there is only a short section about the [Zend compiler](https://www.phpinternalsbook.com/php7/zend_engine/zend_compiler.html) which links to  a great [blog post about the VM](https://www.npopov.com/2017/04/14/PHP-7-Virtual-machine.html). Maybe for now PHP The Right Way wants to link to https://www.npopov.com/2017/04/14/PHP-7-Virtual-machine.html#opcodes.
